### PR TITLE
CSE Docs | Updated standard-match-lists.md

### PR DESCRIPTION
**Pull request by wjakelee**
Updated which match lists are used in which CSE Rules. This was long overdue and initially way out of date.

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by kimsauce** [comment]
+ @ptobia @pkazmir-sumo

File: docs/cse/match-lists-suppressed-lists/standard-match-lists.md [hunk 1]
```diff
@@ -147,20 +114,40 @@ none
...
+* AWS CloudTrail - Aggressive Reconnaissance
+* AWS CloudTrail - Database Snapshot Created
+* AWS CloudTrail - GetSecretValue from non Amazon IP
+* AWS CloudTrail - Reconnaissance related event
+* AWS CloudTrail - sensitive activity in KMS
+* AWS CloudWatch Alarm Actions Disabled
+* AWS CloudWatch Alarm Deletion
+* AWS CloudWatch Anomaly Detector Deletion
+* AWS CloudWatch Log Group Deletion
+* AWS CloudWatch Log Stream Deletion
```
Comment (kimsauce) [inline docs/cse/match-lists-suppressed-lists/standard-match-lists.md:127]: Should be Amazon CloudWatch [per their branding](https://aws.amazon.com/documentation-overview/)

File: docs/cse/match-lists-suppressed-lists/standard-match-lists.md [hunk 2]
```diff
@@ -147,20 +114,40 @@ none
...
+* AWS Image Modification
+* AWS Instance Creation
+* AWS Instance Deletion
+* AWS Instance Discovery
+* AWS Instance Modification
+* AWS Route 53 Domain Registered
+* AWS Route 53 Reconnaissance
+* AWS Route 53 Service Tampering
+* AWS Route 53 TestDNSAnswer
+* AWS Route 53 Traffic Policy Creation
```
Comment (kimsauce) [inline docs/cse/match-lists-suppressed-lists/standard-match-lists.md:144]: Should be **Amazon** Route 53 [per their branding](https://aws.amazon.com/documentation-overview/)

File: docs/cse/match-lists-suppressed-lists/standard-match-lists.md (hunk 1 above)
Comment (wjakelee) [reply docs/cse/match-lists-suppressed-lists/standard-match-lists.md:127]: As John mentioned, these are the names of the Rules in CloudSIEM, so I would recommend our threat labs team changes the names of the rules to align with their branding.

File: docs/cse/match-lists-suppressed-lists/standard-match-lists.md (hunk 2 above)
Comment (wjakelee) [reply docs/cse/match-lists-suppressed-lists/standard-match-lists.md:144]: As John mentioned, these are the names of the Rules in CloudSIEM, so I would recommend our threat labs team changes the names of the rules to align with their branding.

**Comment by kimsauce** [comment]
> @kimsauce, those instances of "AWS" are in standard rule names and therefore can't be changed in the docs.

Oh ok, thank you for clarifying!

**Pull request merged**